docs/people-and-groups/user-provisioning.md

```diff
@@ -0,0 +1,52 @@
+---
+title: User provisioning with SCIM
+---
+
+# User provisioning with SCIM
+
+{% include plans-blockquote.html feature="User provisioning with SCIM" %}
+
+Metabase supports user provisioning via the System for Cross-domain Identity Management (SCIM) protocol. In addition to Single Sign-on (SSO), you can set up user provisioning in Metabase with SCIM to:
+
+- **Decouple authentication from provisioning**. Even though anyone could authenticate with SSO, you may only want some groups to be able to create an account in Metabase.
```

Comment:
I think we should replace groups with users or people here.

It can be confusing with group sync and group mapping